docs/release-notes.md

```diff
@@ -1,5 +1,39 @@
 # Release notes
 
+## Trino Gateway 10 (24 July 2024)
+
+[JAR file gateway-ha-10-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/10/gateway-ha-10-jar-with-dependencies.jar),
+Docker container `trinodb/trino-gateway:10`
```

Comment:
I know that this is outside of the scope for this PR but I think that we should rename docker image to just `trinodb/gateway`

Reply:
no .. we agreed in the past to use Trino Gateway as project and product name consistently. This is also in line with naming things like the Trino CLI, the Trino Go Client and so on..